addons/calendar/models/calendar_event.py

```diff
@@ -1179,22 +1176,13 @@ def ics_datetime(idate, allday=False):
 
         return result
 
-    def convert_online_event_desc_to_text(self, description):
-        """
-        We can sync the calendar events with google calendar, iCal and Outlook, and we
-        also pass the event description along with other data. This description needs
-        to be in plaintext to be displayed properly in above platforms. Because online
-        events have fixed format for the description, this method removes some specific
-        html tags, and converts it into readable plaintext (to be used in external
-        calendars). Note that for regular (offline) events, we simply use the standard
-        `html2plaintext` method instead.
-        """
-        desc_str = str(description)
-        tags_to_replace = ["<ul>", "</ul>", "<li>"]
-        for tag in tags_to_replace:
-            desc_str = desc_str.replace(tag, "")
-        desc_str = desc_str.replace("</li>", "<br/>")
-        return html2plaintext(desc_str)
+    def _get_customer_description(self):
+        """:return (str): The description to include in calendar exports"""
+        return html2plaintext(self.description) or ''
```

Comment:
It is a bit strange but the method html2plaintext returns a string even if we pass False as argument. So if self.description == False, the method will return 'False'. That's why I was suggesting:
```python
return html2plaintext(self.description) if self.description else ''
```

Reply:
Normally it `False or ''` returns `''` so it should work. But I get it's not intuitive, I'll switch to that one :)